packages/documentation/src/stories/components/badge/badge.docs.mdx

```diff
@@ -1,9 +1,21 @@
 import { Canvas, Controls, Meta } from '@storybook/blocks';
+import { PostAlert } from '@swisspost/design-system-components-react';
 import * as badgeStories from './badge.stories';
 import StylesPackageImport from '../../../shared/styles-package-import.mdx';
 
 <Meta of={badgeStories} />
 
+<PostAlert type="warning">
```

Comment:
Nitpick: Just for design coherency, can you please put the alert after the lead like https://next.design-system.post.ch/?path=/docs/components-alert--docs